app/category.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class category extends Model
+{
+    //
+}
```

Comment:
class names are supposed to be CamelCase and also file name

Reply:
category.php was deleted